org.mwc.debrief.core/src/org/mwc/debrief/core/DebriefPlugin.java

```diff
@@ -334,17 +334,17 @@ public void start(final BundleContext context) throws Exception
     ImportReplay.addExtraImporters(importers);
 
     // make Debrief the default editor for XML files
-    Display.getDefault().asyncExec(new Runnable()
-    {
-      
-      @Override
-      public void run()
-      {
-        final IEditorRegistry editorRegistry =
-            PlatformUI.getWorkbench().getEditorRegistry();
-        editorRegistry.setDefaultEditor("*.xml", "org.mwc.debrief.PlotEditor");
-      }
-    });
+//    Display.getDefault().asyncExec(new Runnable()
+//    {
+//      
+//      @Override
+//      public void run()
+//      {
+//        final IEditorRegistry editorRegistry =
+//            PlatformUI.getWorkbench().getEditorRegistry();
+//        editorRegistry.setDefaultEditor("*.xml", "org.mwc.debrief.PlotEditor");
+//      }
+//    });
     

```

Comment:
Hey @saulhidalgoaular  - did I mention how complicated this was?

The above fix does need to be  in the EASE branch - since it's something that breaks under E4.x.

The core develop branch is staying at e3.8.2 (for now) - so the core develop branch still needs the editors to be registered like they are above.